### PR TITLE
 Suppress stderr message when git show command failed

### PR DESCRIPTION
**Pull request by ykitamura-mdsol**
Suppress the following stderr message when the git show command failed:
```
fatal: Not a git repository (or any of the parent directories): .git
```

@jfeltesse-mdsol @jcarres-mdsol @cabbott @ssteeg-mdsol @piao-mdsol

### CODE REVIEW

File: kapnismology.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'mutant-rspec', '~> 0.8'
   s.add_development_dependency 'appraisal', '~> 2.1'
   s.add_development_dependency 'combustion', '~> 1.0'
-  s.add_development_dependency 'sqlite3', '~> 1.3'
+  s.add_development_dependency 'sqlite3', '~> 1.3.6'
```
Comment (ykitamura-mdsol) [inline kapnismology.gemspec]: Add the patch version to solve the sqlite3 version dependency issue:
https://github.com/rails/rails/pull/35154

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
       gemfile: gemfiles/rails_40.gemfile
-    - rvm: 2.5.3
+    - rvm: 2.5.5
+      gemfile: gemfiles/rails_41.gemfile
+    - rvm: 2.6.2
+      gemfile: gemfiles/rails_40.gemfile
+    - rvm: 2.6.2
       gemfile: gemfiles/rails_41.gemfile
 
 gemfile:
```
Comment (jfeltesse-mdsol) [inline .travis.yml]: not in the diff but how about dropping rails 3.x?

**Comment by jfeltesse-mdsol** [comment]
💯

File: .travis.yml (hunk 2 above)
Comment (ykitamura-mdsol) [reply .travis.yml]: dropped in https://github.com/JordiPolo/kapnismology/pull/15/commits/02e128106c7eae68d44b3f8506bc014cfe9dac27

**Pull request merged**